File: isogram/Isogram.cs
using System;

public static class Isogram
{
    public static bool IsIsogram(string word)
    {
        word = word.ToLower();

        foreach (char c in word)
        {
            if (Char.IsLetter(c))
            {
                // find first and last index of the current letter. if they don't match, there are duplicates
                if (word.IndexOf(c) != word.LastIndexOf(c)) 
                {
                    return false;
                }
            }
        }

        return true;
    }
}

File: JavaScript/isogram.js
export const isIsogram = (word) => {
  word = word.toLowerCase();

  // find first and last index of the current letter. if they don't match, there are duplicates
  for (let char of word) {
    if (/[a-z]/.test(char) && (word.indexOf(char) !== word.lastIndexOf(char))) {
      return false;
    }
  }
  return true;
};
